src/by/gsu/epamlab/task2/Material.java

```diff
@@ -21,12 +21,12 @@ public void setName(String name) {
         this.name = name;
     }
 
-    public double getDensity() {
-        return density;
+    public double getDENSITY() {
```

Comment:
you do not need getters for the constants, it is useless. You can reach them by Class or public modificator.